src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -120,6 +125,11 @@ impl<T: AstInfo> AstDisplay for Statement<T> {
             Statement::CreateType(stmt) => f.write_node(stmt),
             Statement::CreateCluster(stmt) => f.write_node(stmt),
             Statement::CreateClusterReplica(stmt) => f.write_node(stmt),
+            Statement::AlterClusterOwner(stmt) => f.write_node(stmt),
```

Comment:
Is it possible to have an `AlterOwner` stmt using your refactored object code?

Reply:
I'll play around with this and see if it's possible. The tricky part is that there are multiple ID types and different ways of resolving the objects.

Reply:
I just pushed something in https://github.com/MaterializeInc/materialize/pull/18402/commits/13d9359d180b655d77860f00994734b1d915bd4d, that cleans this up. Let me know what you think.